qiskit/transpiler/target.py

```diff
@@ -280,6 +280,7 @@ def __init__(
                 matches the qubit number the properties are defined for. If some
                 qubits don't have properties available you can set that entry to
                 ``None``
+            meas_map (list): List of sets of qubits that must be measured together.
```

Comment:
Please revert this file

Reply:
I fixed it at https://github.com/Qiskit/qiskit-terra/pull/9987/commits/3926200fe62bac8bd9be06c06449bc60d3c93224.